app/controllers/users/phone_confirmation_controller.rb

```diff
@@ -0,0 +1,108 @@
+module Users
+  class PhoneConfirmationController < ApplicationController
+    before_action :authenticate_user!
+    before_action :check_for_unconfirmed_mobile
+
+    def show
+      if FeatureManagement.prefill_otp_codes?
+        @code_value = confirmation_code
```

Comment:
According to Code Climate, this line is not tested. I'm using their very nifty browser extension: https://codeclimate.com/browser-extension

You added a similar test in `two_factor_authentication_controller_spec.rb`, but we also need one for this `phone_confirmation_controller`.